src/models/room.js

```diff
@@ -31,7 +31,7 @@ const EventTimelineSet = require("./event-timeline-set");
 
 import ReEmitter from '../ReEmitter';
 
-const LATEST_ROOM_VERSION = '1';
+const KNOWN_SAFE_ROOM_VERSION = '1';
 const SAFE_ROOM_VERSIONS = ['1', '2'];
```

Comment:
A comment on how SAFE_ROOM_VERSIONS and KNOWN_SAFE_ROOM_VERSION feels like it might be useful